hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DefaultCertificateClient.java

```diff
@@ -657,6 +661,9 @@ protected enum InitCase {
    * 2. Generates and stores a keypair.
    * 3. Try to recover public key if private key and certificate is present
    *    but public key is missing.
+   * 4. Checks is the certificate is about to be expired or have already been
```

Comment:
nit: typo in Checks is -> it should be Checks if

Reply:
done